docs/resources/synthetics.md

```diff
@@ -150,6 +159,11 @@ The following arguments are supported:
   - `min_location_failed` - (Optional) Threshold below which a synthetics test is allowed to fail before sending notifications. Default is 1.
   - `accept_self_signed` - (Optional) For type=ssl, true or false
   - `allow_insecure` - (Optional) For type=api, true or false. Allow your HTTP test go on with connection even if there is an error when validating the certificate.
+  - `retry` - (Optional)
+    - `count` - (Optional) Number of retries needed to consider a location as failed before sending a notification alert.
+    - `interval` - (Optional) Interval between a failed test and the next retry in milliseconds.
+  - `monitor_options` - (Optional)
+    - `renotification_interval` - (Optional) Specify a renotification frequency.
```

Comment:
obviously this is long closed but it looks like this is in minutes. I can submit a PR to add that to the docs to help clarify the unit on this number for folks